### PR TITLE
Fixed lockdown

### PR DESCRIPTION
**Pull request by HSalat**
Please check carefully I haven't broken anything

### CODE REVIEW

File: src/init/lockdown.rs [hunk 1]
```diff
@@ -63,11 +63,11 @@ pub fn calculate_lockdown_per_day(
...
     }
 
-    // Find the mean probability of staying at home, over the entire population
-    let mean_pr_home = population
+    // Find the mean decrease of time spent outside of home, over the entire population
+    let mean_pr_home_tot = population
         .people
         .iter()
-        .map(|person| person.time_use.home as f32)
+        .map(|person| person.time_use.home_tot as f32)
```
Comment (dabreegster) [inline src/init/lockdown.rs:70]: Close! I named it home_total: https://github.com/dabreegster/uatk-spc/blob/f4c6a913ad89d604e57664943e4419580a12e5e5/synthpop.proto#L150

**Comment by dabreegster** [comment]
Did you mean to close the PR? The math here makes much more sense to me, there was just a variable name slightly wrong

**Comment by HSalat** [comment]
yeah, I fixed the variable name in the branch, but then the test wouldn't run and I got bored. Otherwise the math and descriptions are correct

**Comment by dabreegster** [comment]
Github is a little slow to re-run tests. All this one does is check that stuff compiles. If it's working in your IDE or with cargo, it should be fine. Thanks for the fix -- merging

**Pull request merged**